platforms/core-configuration/configuration-cache/src/test/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblemsSummaryTest.kt

```diff
@@ -35,15 +35,15 @@ class ConfigurationCacheProblemsSummaryTest {
         val subject = ConfigurationCacheProblemsSummary(maxCollectedProblems = 2)
         assertTrue(
             "1st problem",
-            subject.onProblem(buildLogicProblem("build.gradle", "failure"), ProblemSeverity.Failure)
+            subject.onProblem(buildLogicProblem("build.gradle", "failure"), ProblemSeverity.Failure, false)
```

Comment:
❓ Strong smell. Would it make sense to use a different severity instead?

Reply:
Adressed in the new PR
https://github.com/gradle/gradle/pull/33740/files#diff-edfda9aa7bba771e008c137f006450e990b9d15ee4c05057f2795a686a1e2effR49